### DISCUSSION TITLE
What is editorconfigOverride "android" to "true"?

### DISCUSSION THREAD

**Discussion by Jaehwa-Noh** [diffplug/spotless#2077] [Q&A] [answered]
Sometimes I see this gradle configuration.
What is the meaning of "android" to "true"?
I can't see any documentation but developers use this map.
What is this?
Is there any documentation?

```DSL
ktlint(ktlintVersion).editorConfigOverride(
                    mapOf(
                        "android" to "true",
                    ),
                )
```

**Comment by nedtwigg** [comment, marked as answer]
Spotless is just calling out into ktlint, you'll have to look at their docs

https://github.com/pinterest/ktlint

**Reply by Jaehwa-Noh** [reply]
Thank you. I'll check it.
